File: 2.0/java/webhooks/findWebhooks.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.webhooks.model.ResourceType;
import com.callfire.api.client.api.webhooks.model.Webhook;
import com.callfire.api.client.api.webhooks.model.request.FindWebhooksRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        FindWebhooksRequest request = FindWebhooksRequest.create()
            .name("my webhook")
            .resource(ResourceType.TEXT_BROADCAST)
            .event(ResourceType.ResourceEvent.STARTED)
            .callback("https://myservice/callback")
            .enabled(true)
            .offset(0L)
            .limit(10L)
            .fields("items(id,name,callback)")
            .enabled(true)
            .build();
        Page<Webhook> webhooks = client.webhooksApi().find(request);
    }
}

File: 2.0/csharp/webhooks/findWebhooks.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Webhooks.Model;
using CallfireApiClient.Api.Webhooks.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindWebhooksRequest
        {
            Limit = 5,
            Enabled = true,
            Name = "test_name"
        };
        Page<Webhook> webhooks = client.WebhooksApi.Find(request);
    }
}
